kitsune/users/jinja2/users/auth.html

```diff
@@ -50,7 +49,8 @@ <h1 class="sumo-page-subheading sumo-header-medium">{{ _('Continue with your Moz
                   </p>
                   <div class="trouble-text">
                     <p class="help-text">
-                      {{ _('Having trouble signing in? Contact support' ) }}
+                      {{ _('Having trouble signing in? ')}}
+                      <a href="{{ url('questions.aaq_step3', product_key='mozilla-account') }}">{{ _('Contact support') }}</a>
```

Comment:
Is it `mozilla-account` or `mozilla-accounts`? Currently we have `firefox-accounts`

Reply:
`mozilla-account` - there is one account, not many accounts. There can be only one.